src/atoms/Text/Text.native.js

```diff
@@ -50,14 +39,12 @@ const StyledText = styled(NativeText)`
   font-size: ${styles.fontSize};
   color: ${styles.color};
   text-decoration-line: ${styles.textDecorationLine};
```

Comment:
you didn't had to mention align left as a defualt style as you were mentioning the other day? It takes it by default?

Reply:
The parent will take care